eth-contracts/.solcover.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  providerOptions: {
+    'total_accounts': 20
```

Comment:
nit but why not set this to 50 to be consistent with ganache setup?

also throwing a link to here https://github.com/sc-forks/solidity-coverage#config-options would be helpful for future ref

Reply:
i actually didn't want to run 50 for coverage purposes but it might be worth adding